CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Block cursor is no longer inverted at the start/end of a selection
 - Preserve selection on non-LMB or mouse mode clicks
 - Wayland client side decorations are now based on config colorscheme
+- Windows window icon is now loaded from the embedded resource
```

Comment:
```
- Low resolution window decoration icon on Windows
```

I think this reads a bit better, since `Windows window` is a little awkward. I'd also move this to the fixed section, since from a user perspective it fixes an issue.